src/main/java/app/bpartners/api/endpoint/event/gen/RefreshInvoiceSummaryByUserTriggered.java

```diff
@@ -0,0 +1,17 @@
+package app.bpartners.api.endpoint.event.gen;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.io.Serializable;
+import javax.annotation.processing.Generated;
+import lombok.*;
+
+@Generated("EventBridge")
+@EqualsAndHashCode
+@Builder
+@ToString
+@AllArgsConstructor
+@Data
+public class RefreshInvoiceSummaryByUserTriggered implements Serializable {
```

Comment:
Why not just RefreshUserInvoiceSummaryTriggered ?

Reply:
I already have the RefreshUserInvoiceSummaryTriggered, which is the one which is invoked in the scheduled task in EventBridge